backend/uclapi/uclapi/celery.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import, unicode_literals
+
+import celery
+import os
+import raven
+
+from django.conf import settings
+from raven.contrib.celery import register_signal as raven_register_signal, \
+    register_logger_signal as raven_register_logger_signal
+
+from dotenv import read_dotenv
+
+
+read_dotenv(os.path.join(os.path.dirname(os.path.dirname(__file__)), '.env'))
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'uclapi.settings')
+
+
+class Celery(celery.Celery):
+    def on_configure(self):
+        if os.environ.get("SENTRY_DSN") != "":

```

Comment:
If the environment variable here is not set, that might show a weird error.
Probably best to use `os.environ["SENTRY_DSN"]` ?